marklogic-data-hub-central/ui/src/components/common/hc-card/hc-card.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {Card, CardProps} from "react-bootstrap";
+import styles from "./hc-card.module.scss";
+
+
+export interface HCCardProps extends CardProps {
+  actions?: any[],
+  titleExtra?: any,
```

Comment:
@mrodriguez-endava  @mrodrigu-marklogic 
I think these two should be:
`actions?: Array<JSX.Element>`
`titleExtra?: JSX.Element | null`

Reply:
Weird, I thought I had changed it. `React.ReactNode` works as well? in case you are putting a React component and not an Element?